src/Collapsible/Collapsible.scss

```diff
@@ -1,3 +1,9 @@
+@import "../Card/variables"; // TODO: Remove this dependency
+@import "variables";
+
+// TODO: Remove this dependency
```

Comment:
What's up with these TODOs?

Reply:
Collapsible styles extend Card styles. I think this is the only component with a dependency like this. I would rather we update the component to add card classes or something, but for now figured this got the job done.